action.yaml

```diff
@@ -11,19 +11,25 @@ inputs:
     description: 'Which shell to use; bash by default'
     required: true
     default: 'bash'
+  keyname:
+    description: 'Filename root for SSH key pair'
+    required: false
```

Comment:
I see I used requires True above, but also provided a default. Not sure that even makes sense.